content/applications/finance/fiscal_localizations/argentina.rst

```diff
@@ -948,6 +948,87 @@ the corresponding layout and fiscal stamps stated in the :ref:`Invoice printed r
 .. seealso::
    :doc:`Client account creation <../../websites/ecommerce/checkout_payment_shipping/checkout>`
 
+Liquidity product direct sales
+------------------------------
+
+The liquidity product direct sale is used in cases where sales are made to a third party. In these
+cases, the seller and the proprietary company of the goods can register their corresponding sales
+and purchase. This feature is included when :ref:`installing <general/install>` the *Argentinian
+Electronic Invoicing* module (`l10n_ar_edi`).
+
+Configuration
+~~~~~~~~~~~~~
+
+Document types
+**************
+
+This feature includes the addition of the following document types: 23, 24, 25, 26, 27, 28, 33, 43,
+45, 46, 48, 58, 60, 61, 150, 151, 157, 158, 161, 162, 164, 166, 167, 171, 172, 180, 182, 186, 188,
+332.
```

Comment:
I don't see those documents, but I didn't fully configure my base runbot (17.0) and test db (17.1) aven though I should have installed all the proper modules. Since you mention installing the AR e-invoicing module is required, you could also say it should be configured? Or not mention the installation at all.

Reply:
@xpl-odoo @samueljlieber in this case I think we can avoid mention it at all. We can remove this section talking about the new document types added.